### DISCUSSION TITLE
Why would a sync even delete 84% bookmarks in app?

### DISCUSSION THREAD

**Discussion by sircode** [floccusaddon/floccus#1795] [General]
![Screenshot_2024-12-11-14-47-41.jpg](https://github.com/user-attachments/assets/0a337631-419b-4bd5-bb91-761f73e38cdf)

**Comment by marcelklehr** [comment]
If you deleted a lot of your bookmarks on the other device, this device will refuse to act on that until you disable the failsafe.
This can also happen when you have created a lot of bookmarks in one device and then down-sync from the server where the bookmarks are not, this will delete the local bookmarks.

**Reply by sircode** [reply]
Not sure why this is closed.

Sync is set to "merge". Still I got this warning, that all the bookmarks added in app, would be removed.

**Reply by marcelklehr** [reply]
> Not sure why this is closed.

This is a discussion, if you have an issue with the app you need to create an issue and fill out the issue form. This is so people can help you properly, a simple screenshot is not helpful sadly.
